cms/static/js/views/active_video_upload_list.js

```diff
@@ -162,13 +184,43 @@ define(
                 if (this.onFileUploadDone) {
                     this.onFileUploadDone(this.collection);
                     this.clearSuccessful();
+                    this.fileErrorMsg = null;
                 }
             },
 
             fileUploadFail: function(event, data) {
                 this.setStatus(data.cid, ActiveVideoUpload.STATUS_FAILED);
             },
 
+            showErrorMessage: function(error) {
+                this.fileErrorMsg = new NotificationView.Error({
+                    title: gettext('Your file could not be uploaded'),
+                    message: error
+                });
+                this.fileErrorMsg.show();
```

Comment:
Instead of the setting the `fileErrorMsg` variable on view use a function level variable to show the message and then in tests verify if the error message is present in `DOM` or not. This should work.